src/main/java/se/sundsvall/messaging/api/model/request/SmsRequest.java

```diff
@@ -4,24 +4,30 @@
 
 import java.util.List;
 
-import com.fasterxml.jackson.annotation.JsonIgnore;
-
-import io.swagger.v3.oas.annotations.media.Schema;
 import jakarta.validation.Valid;
 import jakarta.validation.constraints.NotBlank;
-import lombok.Builder;
-import lombok.With;
+import jakarta.validation.constraints.Pattern;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
 import se.sundsvall.dept44.common.validators.annotation.ValidMSISDN;
 import se.sundsvall.dept44.common.validators.annotation.ValidUuid;
 import se.sundsvall.messaging.model.ExternalReference;
 
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.Builder;
+import lombok.With;
+
 @With
 @Builder(setterPrefix = "with")
 public record SmsRequest(
 
 	@Valid @Schema(description = "Party") Party party,
 
-	@Schema(description = "Sender") String sender,
+	@Schema(description = "The sender of the SMS, must be between 3-11 characters and start with a non-numeric character",
+		requiredMode = REQUIRED, maxLength = 11, minLength = 3, example = "sender")
+	@Pattern(regexp = "^[a-zA-Z ][a-zA-Z0-9 ]{2,10}$", message = "sender must be between 3-11 characters and start with a non-numeric character")
```

Comment:
Add `only contain alpha-numeric values` to description?

Reply:
Ok, but shouldn't the description describe that only characters a-z, A-Z, space and 0-9 is allowed. Otherwise clients might think that @, _ and like are allowed, while they are not (if I read the regexp right).